taca/utils/bioinfo_tab.py

```diff
@@ -122,8 +122,14 @@ def update_statusdb(run_dir, inst_brand):
                         # If entry exists, append to existing
                         # Special if case to handle lanes written as int, can be safely removed when old lanes
                         # is no longer stored as int
-                        if len(view[[project, run_id, int(lane), sample]].rows) >= 1:
-                            lane = int(lane)
+                        try:
+                            if (
+                                len(view[[project, run_id, int(lane), sample]].rows)
+                                >= 1
+                            ):
+                                lane = int(lane)
+                        except ValueError:
+                            pass
```

Comment:
Not sure I understand what this does for the `1+2` flowcells, this would just pass on that error?

Reply:
I still don't really get it. The 1+2 would not be possible to cast to an integer and therefore be ignored?

Reply:
Ok, I think I get it now. Thank you!